### PR TITLE
Added light assigner unit tests and unittest framework

### PR DESCRIPTION
**Pull request by brhoades**
Added light unit tests for assigner (assigner/__init__.py) and some framework for unittest.

Part of #28.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ Pull requests are welcome! If you have questions, feel free to [contact the auth
 Before merging:
 
 - Describe the change in `CHANGELOG.md`
+- Create supporting unit tests.
+- Verify new and existing unit tests pass with `python setup.py nosetests`.
```
Comment (LinuxMercedes) [inline CONTRIBUTING.md]: Should this be `python setup.py test` ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,9 @@
 - Removed remaining lint as specified by pylint.
 - Removed old baserepo standalone code.
 - Added Travis CI config for pylint and pyflakes.
+- Added unittest scaffolding with nose
+- Added `AssignerTestCase`
```
Comment (LinuxMercedes) [inline CHANGELOG.md]: Let's merge this with the line above?

File: CONTRIBUTING.md (hunk 1 above)
Comment (brhoades) [reply CONTRIBUTING.md]: Unfortunately, even with the test runner change I made in setup.py, I couldn't get that command to use nose.

Comment (brhoades) [reply CONTRIBUTING.md]: `setup.py` test runs unittest

Comment (LinuxMercedes) [reply CONTRIBUTING.md]: So when I do `python setup.py nosetest` I get `error: invalid command 'nosetest'`...

Comment (brhoades) [reply CONTRIBUTING.md]: :/
Did you do `pip install -r requirements.txt`?

File: CHANGELOG.md (hunk 2 above)
Comment (brhoades) [reply CHANGELOG.md]: Done.

File: CONTRIBUTING.md (hunk 1 above)
Comment (LinuxMercedes) [reply CONTRIBUTING.md]: Whoops, its `nosetests`, not  `nosetest`. I think I tried the former earlier but that was before I made a new virtualenv for it.

**Pull request merged**